lib/workers/repository/config-migration/branch/commit-message.spec.ts

```diff
@@ -23,7 +23,7 @@ describe('workers/repository/config-migration/branch/commit-message', () => {
       fileName,
     );
     expect(commitMessageFactory.getPrTitle()).toBe(
-      'chore(config): migrate renovate config',
+      'chore(config): migrate Renovate config',
```

Comment:
This change follows what we intend our Semantic commit message capitalisation to do https://github.com/renovatebot/renovate/blob/f33076854f017ee2d247fc29921fd56da6ac8aaa/lib/workers/repository/model/semantic-commit-message.spec.ts#L28-L35